<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPrincipal</class>
 <widget class="QWidget" name="VentanaPrincipal">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VentanaPrincipal</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
   border-image: url(:/images/myimage.png) 0 0 0 0 stretch stretch;
   border-width: 0px;
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>791</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#inicio
{
   border-image: url(:/img/portada.jpeg) 0 0 0 0 stretch stretch;
}
QFrame
{
background-color: rgb(255, 255, 255);
}
QPushButton#reloadButton
{
   border-image: url(:/img/reload.png) 0 0 0 0 stretch stretch;
}</string>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="inicio">
    <widget class="QPushButton" name="comenzarButton">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>390</y>
       <width>211</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Schoolbook L</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Comenzar</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="entrada">
    <widget class="QPushButton" name="submit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>536</y>
       <width>421</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Iniciar Juego</string>
     </property>
    </widget>
    <widget class="QFrame" name="horizontalFrame_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>400</y>
       <width>421</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Ingresar puerto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="port"/>
      </item>
     </layout>
     <zorder>port</zorder>
     <zorder>label_3</zorder>
    </widget>
    <widget class="QFrame" name="horizontalFrame_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>421</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ingresar host</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="host"/>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="horizontalFrame">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>421</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame 
{
background-color: rgb(255, 255, 255);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>25</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Ingresar Nombre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nombre"/>
      </item>
     </layout>
     <zorder>nombre</zorder>
     <zorder>label</zorder>
    </widget>
    <zorder>horizontalFrame_3</zorder>
    <zorder>horizontalFrame_2</zorder>
    <zorder>horizontalFrame</zorder>
    <zorder>submit</zorder>
   </widget>
   <widget class="QWidget" name="eleccionSala">
    <widget class="QPushButton" name="crearSalaButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>520</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Crear Sala</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elegirSalaButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>440</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Elegir Sala</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listaSalas">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>301</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Salas Disponibles</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="reloadButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>260</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="crearSala">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>741</width>
       <height>651</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>116</x>
        <y>16</y>
        <width>521</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ingrese el nombre de la sala</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombreSala">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="crearSalaButton_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>361</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="salaEspera">
    <widget class="QLabel" name="integrantesLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>411</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Integrantes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sala">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>411</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sala Nombre</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="comenzarJuego">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>500</y>
       <width>301</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Comenzar</string>
     </property>
    </widget>
    <widget class="QListWidget" name="integrantesLista">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>411</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="reloadButton_2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>200</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
